src/SFA.DAS.PSRService.Domain/Entities/Report.cs

```diff
@@ -122,61 +124,15 @@ private ReportingPercentages GetPercentages()
+                TotalEmployees += Convert.ToInt32(employmentEnd);
 
-            if (ReportingPercentages != null)
+            if (ReportingPercentages != null && employeesSection == "YourEmployees" && apprenticesSection == "YourApprentices")
                 percentages.Title = ReportingPercentages.Title;
 
-            return percentages;
-
-        }
-        private ReportingPercentages GetPercentagesSchools()
-        {
-            var percentages = new ReportingPercentages();
-
-            if (Sections == null)
-                return null;
-
-            var employeeQuestions = GetQuestionSection("SchoolsEmployees");
-
-            if (employeeQuestions == null)
-                return null;
-
-            var apprenticeQuestions = GetQuestionSection("SchoolsApprentices");
-
-            if (apprenticeQuestions == null)
-                return null;
-
-
-            decimal employmentPeriod = 0, apprenticePeriod = 0, employmentEnd = 0, apprenticeEnd = 0, employmentStart = 0;
-
-            if (String.IsNullOrWhiteSpace(employeeQuestions.Questions.Single(w => w.Id == "newThisPeriod").Answer) == false)
-                employmentPeriod = decimal.Parse(employeeQuestions.Questions.Single(w => w.Id == "newThisPeriod").Answer);
-            if (String.IsNullOrWhiteSpace(apprenticeQuestions.Questions.Single(w => w.Id == "newThisPeriod").Answer) == false)
-                apprenticePeriod = decimal.Parse(apprenticeQuestions.Questions.Single(w => w.Id == "newThisPeriod").Answer);
-
-            if (String.IsNullOrWhiteSpace(employeeQuestions.Questions.Single(w => w.Id == "atEnd").Answer) == false)
-                employmentEnd = decimal.Parse(employeeQuestions.Questions.Single(w => w.Id == "atEnd").Answer);
-            if (String.IsNullOrWhiteSpace(apprenticeQuestions.Questions.Single(w => w.Id == "atEnd").Answer) == false)
-                apprenticeEnd = decimal.Parse(apprenticeQuestions.Questions.Single(w => w.Id == "atEnd").Answer);
-
-            if (String.IsNullOrWhiteSpace(employeeQuestions.Questions.Single(w => w.Id == "atStart").Answer) == false)
-                employmentStart = decimal.Parse(employeeQuestions.Questions.Single(w => w.Id == "atStart").Answer);
-
-
-            if (apprenticePeriod != 0 & employmentPeriod != 0)
-                percentages.EmploymentStarts = ((apprenticePeriod / employmentPeriod) * 100).ToString("F2");
-
-            if (apprenticeEnd != 0 & employmentEnd != 0)
-                percentages.TotalHeadCount = ((apprenticeEnd / employmentEnd) * 100).ToString("F2");
-
-            if (apprenticePeriod != 0 & employmentStart != 0)
-                percentages.NewThisPeriod = ((apprenticePeriod / employmentStart) * 100).ToString("F2");
-
-            percentages.Title = ReportingPercentagesSchools.Title;
+            if (ReportingPercentagesSchools != null && employeesSection == "SchoolsEmployees" && apprenticesSection == "SchoolsApprentices")
```

Comment:
can we avoid using magic strings everywhere?

Reply:
Also the title property could be set above this level so that this check isn't required